privacyidea/lib/apps.py

```diff
@@ -81,14 +98,18 @@ def create_google_authenticator_url(key=None, user=None,
                                     realm=None, tokentype="hotp", period=30,
                                     serial="mylabel", tokenlabel="<s>",
                                     hash_algo="SHA1", digits="6",
-                                    issuer="privacyIDEA", user_obj=None):
+                                    issuer="privacyIDEA", user_obj=None,
+                                    extra_data=None):
     """
     This creates the google authenticator URL.
     This url may only be 119 characters long.
     If the URL would be longer, we shorten the username
 
     We expect the key to be hexlified!
     """
+    if extra_data is None:
+        extra_data = {}
```

Comment:
Alright,  815577b943a9090ec4211bfac7bf8655130c3a31 changes this. I used to use the `` if extra_data is None:`` stuff because the ``extra_data or {}`` approach creates a new dictionary in case ``extra_data == {}``... but that's a microoptimization :)